util/tracing/shadow.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracing
+
+import (
+	"github.com/lightstep/lightstep-tracer-go"
+	"github.com/opentracing/opentracing-go"
+	"golang.org/x/net/context"
+)
+
+type shadowTracerManager interface {
+	Name() string
+	Close(tr opentracing.Tracer)
+}
+
+type lightStepManager struct{}
```

Comment:
maybe we can make lightstep to other standalone package and using pattern like `sql` & `github.com/go-sql-driver/mysql`'s import package way to link them up?